docs/process/guidance/feature_flags/index.rst

```diff
@@ -0,0 +1,507 @@
+      or touch CI configuration in order to configure their feature flags and
+      respective build/test configuration.
+
+2. Avoiding duplicated build/test steps: We should avoid wastefulness when
+   running our builds / tests. If we are running two sets of tests, with a feature
+   enabled/disabled, an application/component or test that is not affected by the
+   configuration switch should not need to be built/executed again. This poses
+   some interesting challenges that will be detailed later in this document.
+
+3. Avoid validation complexity: Feature-flagged systems will inherently make
+   our integration processes more complex, especially regarding testing, since now
+   multiple code paths for the same product must be tested. This can be aggravated
+   with combinatorial configurations which can quickly explode the number of test
+   cases. Different levels of abstraction are also affected by different
+   restrictions, for unit testing it might be trivial to test all features but at
+   integration level it might be impossible to test all feature sets due to the
+   inherent complexity and resource limitations.
+
+   1. To avoid combinations, features **should** be independent of each other.
+      If there is a combination of settings that represent a feature, it might
+      make sense to bring them all together in a single flag.
+
+   2. To start off, we test two configurations:
+
+      - Configuration of the Core Software Stack
+      - Configuration for increased coverage of features
+
+      This restricts feature flags to at most two tested values.
+      It is recommended to not use more values for a feature flag.
+      On request, further configurations may be added after impact evaluation.
+
+   3. Features flags should be treated as inventory with a carrying cost.
+      In order to keep the number of feature flags manageable, it is recommended to
+      time-scope flags and teams must be proactive in its management. One
+      possible approach is to create an "expiration date" or "cleanup" task when
+      adding new flags. The responsible group for governance shall supervise this
+      practice but responsibility for maintenance must remain with the area.
+
+Feature flags structure
+-----------------------
+
+SCORE has four categories of feature flags.
+
+Generic requirements for feature flags:
+
+- Any feature flag shall be defined with a reasonable default value.
+- Features that are experimental must be clearly marked as such by prepending `experimental_`
+  (e.g. `experimental_<flag_name>`).
+- Above every feature flag in the BUILD file a comment must link to the relevant documentation.
+
+Below, we list the four categories of feature flags and provide additional information and constraints for each.
+
+Enabling a feature
+^^^^^^^^^^^^^^^^^^
+
+This feature flag is unique per feature in the feature tree and mentioned in the feature description at the top of the
```

Comment:
Are we talking about score-features here? What is a "feature tree"? The transition from feature flags to score features is rather confusing, as mentioned before.

Reply:
Feature tree is what we coined the directory tree under https://github.com/eclipse-score/score/tree/main/docs/features

See my naming argument above.